class_files/words.html

```diff
@@ -62,7 +70,8 @@ <h1>Seven Dwarfs <span>Guessing Game</span></h1>
 
         while (noAnswer) {
           // guess = guess.toLowerCase();
-          if (guess == "doc" || guess == "grumpy" || guess == "sleepy" || guess == "bashful" || guess == "dopey" || guess == "happy" || guess == "sneezy") {
+          if (guess == "Doc" || guess == "Grumpy" || guess == "Sleepy" || guess == "Bashful" || guess == "Dopey" || guess == "Happy" || guess == "Sneezy") {
```

Comment:
Add this commented outline so the class can see why the change is needed when I do the demo.